udata/commands/fixtures.py

```diff
@@ -30,11 +30,21 @@
 DEFAULT_FIXTURE_FILE = (
     "https://raw.githubusercontent.com/opendatateam/udata-fixtures/main/results.json"  # noqa
 )
+DEFAULT_FIXTURES_RESULTS_FILE = "results.json"
+
+
+def fix_dates(obj):
+    """Fix dates from the fixtures so they can be safely reloaded later on."""
+    obj["created_at_internal"] = obj["internal"]["created_at_internal"]
+    obj["last_modified_internal"] = obj["internal"]["last_modified_internal"]
+    del obj["internal"]
+    del obj["created_at"]
 
 
 @cli.command()
 @click.argument("data-source")
-def generate_fixtures_file(data_source):
+@click.argument("results-file", default=DEFAULT_FIXTURES_RESULTS_FILE)
+def generate_fixtures_file(data_source, results_file):
```

Comment:
I would type hint the args and output of the function to comply with the rest of the code which is mostly type hinted  or aiming to be fully type hinted. In this function for example, it could really help understanding it for a newbie of this code in a blink of an eye. But maybe that's because I'm a type hint nazi :)

Like 

```python
def generate_fixtures_file(data_source: str, results_file: str) -> None:
    ...
```